Components/Config.php

```diff
@@ -362,6 +362,12 @@ public function setImportConfigs($data)
                     $model->setValue($configValue);
                 }
                 $this->manager->persist($model);
+
+                if ($key === 'importImagesOnFirstImport' && $configValue === 0) {
```

Comment:
what about export products with cronjob when cron plugin is not active? This case is described in jira.

Reply:
oh fuck i somehow missed that completely 
I will add that functionality for update Products too